test/tbb/test_task_group.cpp

```diff
@@ -258,16 +258,21 @@ void TestThreadSafety() {
     // Test and warm up allocator.
     tests();
 
-    // Ensure that cosumption is stabilized.
-    std::size_t initial = utils::GetMemoryUsage();
-    for (;;) {
+#if !__TBB_USE_ADDRESS_SANITIZER
+    constexpr size_t num_trials = 1000;
+    bool no_memory_leak = false;
```

Comment:
I am Ok to disable the test case for ASAN; however, the limited number of trails might lead to test instability in some corner cases. Can we remain it endless while disabling with ASAN?